precompile/params.go

```diff
@@ -0,0 +1,28 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package precompile
+
+import "github.com/ethereum/go-ethereum/common"
+
+// Gas costs for stateful precompiles
+const (
+	ModifyAllowListGasCost = 20_000
```

Comment:
nit: move to `allow_list.go`?

Reply:
This is intended to follow the pattern of putting the gas costs in a params files as the evm does with the params package. If you feel strongly about moving it to allow list, I can move it though.

Reply:
Being consistent is most important.